tpdatasrc/tpgamefiles/scr/tpModifiers/sp_appraising touch.py

```diff
@@ -0,0 +1,23 @@
+from templeplus.pymod import PythonModifier
+from toee import *
+import tpdp
+from utilities import *
+import spell_utils
+print "Registering sp-Appraising Touch"
+
+def appraisingTouchSpellBonusToAppraise(attachee, args, evt_obj):
```

Comment:
Yknow, in ToEE this spell is pretty busted. Appraise has a wildly different meaning in ToEE than in P&P, direcrly translating to thousands of GPs or more.

Reply:
I have no problem changing spells for balacing reasons.  Change it to  +1/casterlevel?

Reply:
I think it's better to leave it out completely, since this starts getting too much into homebrew territory for my liking, which is better left for 3rd party mods.

Reply:
Ok I will drop it then